src/generators/operationGenerator.ts

```diff
@@ -1026,7 +1038,6 @@ export function addOperationSpecs(
     operation =>
       hasMediaType(operation, KnownMediaType.Json) ||
       hasMediaType(operation, KnownMediaType.Form) ||
-      hasMediaType(operation, KnownMediaType.Binary) ||
```

Comment:
Normally, in our swaggers anything set to binary, the mediaType is set to XML. So, we need not have this condition here.